services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/pdf/HaittojenhallintasuunnitelmaPdfEncoder.kt

```diff
@@ -77,105 +81,164 @@ class HaittojenhallintasuunnitelmaPdfEncoder(private val mapGenerator: MapGenera
             val worstIndexes = kaivuilmoitusalue.worstCasesInTormaystarkastelut()
 
             document.section(hankealue.nimi) {
-                row("Toimet työalueiden haittojen hallintaan, ${hankealue.nimi}", toimetTitle())
-                row("", nuisanceScore(worstIndexes?.liikennehaittaindeksi?.indeksi))
+                hankealueRow(
```

Comment:
Could all these sections be in separate functions in order to shorten the `formatPdf` function?

Reply:
They could, but there would be so many parameters, that the function wouldn't be much shorter.